Emby.Server.Implementations/Library/UserManager.cs

```diff
@@ -216,10 +216,10 @@ public void Initialize()
 
         public static bool IsValidUsername(string username)
         {
-            //This is some regex that matches only on unicode "word" characters, as well as -, _ and @
-            //In theory this will cut out most if not all 'control' characters which should help minimize any weirdness
+            // This is some regex that matches only on unicode "word" characters, as well as -, _ and @
+            // In theory this will cut out most if not all 'control' characters which should help minimize any weirdness
             // Usernames can contain letters (a-z + whatever else unicode is cool with), numbers (0-9), dashes (-), underscores (_), apostrophes ('), and periods (.)
```

Comment:
Can you mention that "@" is a valid character in this comment? (And also the error message below).

Reply:
Yes please! I'm happy that this will finally work so I can use full email addresses for usernames.

Reply:
// Usernames can contain letters (a-z + whatever else unicode is cool with), numbers (0-9), at-signs (@), dashes (-), underscores (_), apostrophes ('), and periods (.)